app/src/main/java/net/sourceforge/opencamera/SyncSettingsContainer.java

```diff
@@ -90,6 +64,35 @@ public SyncSettingsContainer(boolean syncISO, boolean syncWb, boolean syncFlash,
         this.format = format;
     }
 
+    /**
+     * Builds {@link SyncSettingsContainer} using settings collected from the current device.
+     *
+     * @param mainActivity - MainActivity of this device.
+     */
+    public static SyncSettingsContainer build(MainActivity mainActivity) {
```

Comment:
I think that build as a name looks strange, it is so because we know builders that looks like a.setX(x).setY(y).build()
So in my opinion it will be better to use fromActivity(), from(), constructFrom(), buildFrom() then just "build"